### PR TITLE
Web Apps

### PR DESCRIPTION
**Pull request by zobe123**
- added iOS WebAPP Icons/Splashscreens
- added android WebAPP Icons/Splashscreens
- added IE10 Icons
- Updated plexpy.css - prevents the text is larger than the box on small screens.

### CODE REVIEW

**Comment by Hellowlol** [comment]
lol awesome title

File: data/interfaces/default/base.html [hunk 1]
```diff
@@ -20,12 +20,130 @@
...
+
+    <!-- ICONS -->
+    <!-- IE10 icon --> 
+    <meta name="PlexPy" content="PlexPy"/>
+    <meta name="msapplication-TileColor" content="#000000"/>
+    <meta name="msapplication-square70x70logo" content="tiny.png"/>
+    <meta name="msapplication-square150x150logo" content="square.png"/>
+    <meta name="msapplication-wide310x150logo" content="wide.png"/>
+    <meta name="msapplication-square310x310logo" content="large.png"/>
+    <!-- Android >M39 icon -->
```
Comment (JonnyWong16) [inline data/interfaces/default/base.html:41]: Can we use a manifest.json instead? See: https://developers.google.com/web/updates/2015/10/splashscreen?hl=en

Use this:

```
<link rel="manifest" href="interfaces/default/json/Android-manifest.json">
```

Save this file to `interfaces/default/json/Android-manifest.json` and add the referenced icons.

```
{
  "name": "PlexPy",
  "display": "standalone",
  "icons": [
    {
      "src": "../images/res/android/icon-192x192.png",
      "sizes": "192x192",
      "type": "image/png"
    },
    {
      "src": "../images/res/android/icon-256x256.png",
      "sizes": "256x256",
      "type": "image/png"
    },
    {
      "src": "../images/res/android/icon-384x384.png",
      "sizes": "384x384",
      "type": "image/png"
    },
    {
      "src": "../images/res/android/icon-512x512.png",
      "sizes": "512x512",
      "type": "image/png"
    }
  ],
  "background_color": "#1F1F1F",
  "theme_color": "#000000"
}
```

File: data/interfaces/default/base.html [hunk 2]
```diff
@@ -20,12 +20,130 @@
...
+    <link rel="icon" sizes="48x48"href="interfaces/default/images/res/android/48.png">
+    <link rel="icon" sizes="72x72"href="interfaces/default/images/res/android/72.png">
+    <link rel="icon" sizes="96x96"href="interfaces/default/images/res/android/96.png">    
+    <link rel="icon" sizes="144x144"href="interfaces/default/images/res/android/144.png">
+    <link rel="icon" sizes="192x192"href="interfaces/default/images/res/android/192.png">
+    <link rel="icon" sizes="512x512"href="interfaces/default/images/res/android/512.png">
+    <!-- iPad retina icon -->
```
Comment (JonnyWong16) [inline data/interfaces/default/base.html:43]: Do we really need this many icons for iOS?
See "Web clip icon (recommended for web apps and websites)":
https://developer.apple.com/library/ios/documentation/UserExperience/Conceptual/MobileHIG/IconMatrix.html#//apple_ref/doc/uid/TP40006556-CH27

File: data/interfaces/default/base.html [hunk 3]
```diff
@@ -20,12 +20,130 @@
...
+          content="black">
+    <!-- Set the viewport. -->
+    <meta name="viewport"
+          content="initial-scale=1">
+    <!-- Disable automatic phone number detection. -->
+    <meta name="format-detection"
+          content="telephone=no">
+
+    <!-- ICONS -->
+    <!-- IE10 icon -->
```
Comment (JonnyWong16) [inline data/interfaces/default/base.html:37]: Let's also use `xml` for IE. https://msdn.microsoft.com/library/hh772707(v=vs.85).aspx

```
<meta name="application-name" content="PlexPy" />
<meta name="msapplication-config" content="interfaces/default/xml/IEconfig.xml"/>
```

Save this file to `interfaces/default/xml/IEconfig.xml`.

```
<?xml version="1.0" encoding="utf-8"?>
<browserconfig>
  <msapplication>
    <tile>
      <square70x70logo src="interfaces/default/images/res/ie10/tiny.png"/>
      <square150x150logo src="interfaces/default/images/res/ie10/square.png"/>
      <wide310x150logo src="interfaces/default/images/res/ie10/wide.png"/>
      <square310x310logo src="interfaces/default/images/res/ie10/large.png"/>
      <TileColor>#1F1F1F</TileColor>
    </tile>
  </msapplication>
</browserconfig>
```

**Comment by JonnyWong16** [comment]
Can you also add this to `.body-container` in `plexpy.css`? (#437)

```
overflow-y: scroll;
-webkit-overflow-scrolling: touch;
```

**Comment by JonnyWong16** [comment]
Also, can you make the Android/iOS icons with just the golden circle without any text?

File: data/interfaces/default/base.html (hunk 2 above)
Comment (zobe123) [inline data/interfaces/default/base.html:43]: To specify multiple icons for different device resolutions it's necessary to add These lines. (i think so)

**Comment by zobe123** [comment]
Sure, we can do this... but it Looks great as it is. Look yourself:
img_0107

**Comment by JonnyWong16** [comment]
Hmm. I was thinking it should say "PlexPy" under the icon instead of the server name.

**Comment by zobe123** [comment]
Well, it does but you can change it to whatever you want. btw, it doesn't matter, take a look on the IMDb logo...

**Comment by JonnyWong16** [comment]
Ok, looks good.

I tested on Android and I trust that you tested on iOS.

**Comment by zobe123** [comment]
I tested, but it actually opens safari when you hit a menu item... :/
And scrolling differs from the regular view.

**Comment by adamcnaglich** [comment]
I'm not too familiar with GitHub - Can I add css attributes or do I need to make a request? I'm interested in adding -webkit-overflow-scrolling: touch; to the .body-container class in the stylesheet for smoother scrolling on iOS.

Thanks,
Adam

**Comment by Hellowlol** [comment]
For simple things like that I would have just browsed the files of the project on github, hit the edit file button. Do your stuff save and send a pr.

**Comment by adamcnaglich** [comment]
Done. Thanks!

**Comment by JonnyWong16** [comment]
We already included it in this PR.

https://github.com/drzoidberg33/plexpy/blob/dev/data/interfaces/default/css/plexpy.css#L2473

**Comment by adamcnaglich** [comment]
Great... I was excited for my first commit lol.

**Pull request merged**